index.js

```diff
@@ -95,6 +95,12 @@ module.exports.options = {
     watch: {
         default: false,
         runtimeParameter: 'watch'
+    },
+    serializers: {
+        marks: {
+            annotations: [],
+        },
+        types: {},
```

Comment:
@rudietuesdays please note that the format of exported options might be different from how you implemented it. 
Here is the docs for the options: https://github.com/stackbit/sourcebit/wiki/Plugin-API#options-object
The value of every option is not their defaults, but an object with specific properties, one of which can be `default` with the default value.